package.json

```diff
@@ -124,6 +124,7 @@
     "core-js": "^2.5.1",
     "create-plunker": "^1.4.0",
     "css-loader": "^0.28.9",
+    "ejs": "^2.5.7",
```

Comment:
This shouldn't be needed - are you running the latest version of npm? If you copy the package-lock.json from master and `rm -rf node_modules` first before doing `npm i` it should do the trick.

Reply:
I checked out the project from scratch and did npm I. When running the tests it complained that ejs was missing. I'll give it another try and update my npm to latest.